core.yml

```diff
@@ -4,6 +4,7 @@
 require:
   - rubocop-performance
   - rubocop-rspec
+  - rubocop-inflector
```

Comment:
Since this requires `ActiveSupport::Inflector`, would it make more sense to be in the `rails.yml` configuration rather than the `core.yml` configuration?